python-sdk/noxfile.py

```diff
@@ -21,7 +21,7 @@ def dev(session: nox.Session) -> None:
 
 
 @nox.session(python=["3.7", "3.8", "3.9"])
-@nox.parametrize("airflow", ["2.2.5", "2.3"])
+@nox.parametrize("airflow", ["2.2.5", "2.3", "2.4.0b1"])
```

Comment:
I am just trying it with 2.4.0b1 -- and see if we can reproduce the error

Reply:
wierd, because i tried in morning with 2.4.0b1 and didnt get this error

Reply:
but you didnt change `pyproject.toml` ?